starter-core/src/main/java/io/micronaut/starter/feature/crac/Crac.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.feature.crac;
+
+import io.micronaut.core.annotation.NonNull;
+import io.micronaut.starter.application.ApplicationType;
+import io.micronaut.starter.application.generator.GeneratorContext;
+import io.micronaut.starter.build.dependencies.MicronautDependencyUtils;
+import io.micronaut.starter.build.gradle.GradlePlugin;
+import io.micronaut.starter.feature.Category;
+import io.micronaut.starter.feature.Feature;
+import jakarta.inject.Singleton;
+
+@Singleton
+public class Crac implements Feature {
+
+    public static final String FEATURE_NAME_GRAALVM = "crac";
+
+    @Override
+    @NonNull
+    public String getName() {
+        return FEATURE_NAME_GRAALVM;
+    }
+
+    @Override
+    public String getTitle() {
+        return "Support for CRaC (Coordinated Restore at Checkpoint)";
+    }
+
+    @Override
+    public boolean isPreview() {
+        return true;
+    }
+
+    @Override
+    @NonNull
+    public String getDescription() {
+        return "Allows building an application that supports CRaC (Coordinated Restore at Checkpoint)";
+    }
+
+    @Override
+    public String getThirdPartyDocumentation() {
+        return "https://wiki.openjdk.org/display/CRaC";
+    }
+
+    @Override
+    public String getMicronautDocumentation() {
+        return "https://micronaut-projects.github.io/micronaut-crac/latest/guide";
+    }
+
+    @Override
+    public boolean supports(ApplicationType applicationType) {
+        return applicationType == ApplicationType.DEFAULT || applicationType == ApplicationType.CLI;
+    }
+
+    @Override
+    public String getCategory() {
+        return Category.OTHER;
```

Comment:
I don't think Other is a valid category

Reply:
https://github.com/micronaut-projects/micronaut-starter/blob/a5f4cbd0420ee9774dfd12e8a36aad61f0b462d9/starter-core/src/main/java/io/micronaut/starter/feature/Category.java#L41

Or do you mean it's not the correct category?  🤔  I couldn't find one that matched up...  PACKAGING?

Reply:
Yes, same as the Graal feature: 

https://github.com/micronaut-projects/micronaut-starter/blob/3.6.x/starter-core/src/main/java/io/micronaut/starter/feature/graalvm/GraalVM.java#L55

Reply:
> https://github.com/micronaut-projects/micronaut-starter/blob/a5f4cbd0420ee9774dfd12e8a36aad61f0b462d9/starter-core/src/main/java/io/micronaut/starter/feature/Category.java#L41
> 
> Or do you mean it's not the correct category? 🤔 I couldn't find one that matched up... PACKAGING?

I mean we have a test to ensure you don't use OTHER

https://github.com/micronaut-projects/micronaut-starter/blob/3.6.x/starter-core/src/test/groovy/io/micronaut/starter/feature/CategorySpec.groovy#L11